client/src/locales/en-US.json

```diff
@@ -1013,7 +1023,9 @@
             "actionDetails": "View details",
             "titleAssignRoles": "Copy roles",
             "actionAssignRoles": "Copy workspace roles to...",
-            "subtitleAssignRoles": "You can copy {sourceUser} roles to another person. Workspace roles will not be removed."
+            "subtitleAssignRoles": "You can copy {sourceUser} roles to another person. Workspace roles will not be removed.",
+            "titleUpdateProfile": "Profile",
+            "actionUpdateProfile": "Update the profile | Update the profiles"
```

Comment:
Selecting multiple users and opening the context menu only affects one user, I don't think this plural is being used. But it might be worth opening an issue for multiple selection context menu

Reply:
Will be adressed by #9474